frontend/src/components/footerPurple/FooterPurpleElements.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const FooterContainer = styled.footer`
+  display: flex;
+  background: linear-gradient(
+    88.54deg,
+    #6930c3 -0.21%,
```

Comment:
(optional) In company projects we try to create variables for such values. In most reviews ppl would ask "why -0.21%" :)